### ISSUE TITLE
Some answers are not correct

### ISSUE THREAD

**Issue by lxtGH** [JudasDie/deeplearning.ai#1]
I see some answers are not correct

**Comment by JudasDie** [comment]
Yes, some demo maybe not so correct, but some are initialization issues.  Be free to correct yourself. Thank you for your comment.

**Comment by yashtrivedi96** [comment]
In week4/ Face Recognition assignment, the answer for the triplet_loss is incorrect. 
Try axis = -1 for step 1 and 2
 # Step 1: Compute the (encoding) distance between the anchor and the positive, you will need to sum over axis=-1
    pos_dist = tf.reduce_sum(tf.square(tf.subtract(anchor, positive)), axis = -1)
    # Step 2: Compute the (encoding) distance between the anchor and the negative, you will need to sum over axis=-1
    neg_dist = tf.reduce_sum(tf.square(tf.subtract(anchor, negative)), axis = -1)
    # Step 3: subtract the two previous distances and add alpha.
    basic_loss = tf.add(tf.subtract(pos_dist, neg_dist), alpha)
    # Step 4: Take the maximum of basic_loss and 0.0. Sum over the training examples.
    loss = tf.reduce_sum(tf.maximum(basic_loss, 0 ))
